README.md

```diff
@@ -64,6 +64,10 @@ We support using exponentjs to run, debug and publish applications. For more inf
 
 For more details about configuring and debugging Expo applications see [Expo docs](doc/expo.md)
 
+## Build APK and Generate Bundle
+
+You can add VSCode tasks for build apk and generate iOS/Android bundle. See [here](tasks.md) for more info.
```

Comment:
I would change this to: "You can add VSCode tasks to build an .apk file and generate iOS/Android bundles"

Reply:
changed here
https://github.com/Microsoft/vscode-react-native/pull/554/commits/b16e0e9cab895d50afe5b9fde42de832b301e6d6